packages/grpc-native-core/ext/node_grpc.cc

```diff
@@ -252,6 +253,7 @@ NAN_METHOD(SetDefaultLoggerCallback) {
     grpc_logger_state.logger_set = true;
   }
   grpc_logger_state.callback = new Nan::Callback(info[0].As<v8::Function>());
+  grpc_logger_state.async_resource = new Nan::AsyncResource("grpc:logger");
```

Comment:
I didn't find a delete of `grpc_logger_state.callback`, but I didn't look too hard either. Wherever we free one, we should free the other.